python_src/tests/ingestion/test_gtfs_rt_converter.py

```diff
@@ -165,8 +156,17 @@ def test_vehicle_positions_file_conversion() -> None:
     assert timestamp.year == 2019
     assert timestamp.day == 12
 
-    assert table.shape[0] > 0
-    assert table.shape[1] > 0
+    np_df = converter.detail.flatten_schema(table).to_pandas()
+    np_df = drop_list_columns(np_df)
+
+    parquet_file = os.path.join(
+        test_files_dir, "ingestion_GTFS-RT_VP_OLD.parquet"
+    )
+    parquet_df = pandas.read_parquet(parquet_file)
+    parquet_df = drop_list_columns(parquet_df)
+
+    compare_result = np_df.compare(parquet_df, align_axis=1)
+    assert compare_result.shape[0] == 0, f"{compare_result}"
```

Comment:
Change these tests to do whole table compares from parquet files instead of just checking max/min values that are hard coded.